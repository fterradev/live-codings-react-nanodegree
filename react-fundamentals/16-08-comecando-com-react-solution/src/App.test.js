import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

global.localStorage = {
  getItem: () => {}
}

it('renders without crashing', () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
