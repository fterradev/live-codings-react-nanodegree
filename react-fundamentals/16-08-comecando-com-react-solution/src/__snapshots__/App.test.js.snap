// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h2>
      To Do List
    </h2>
  </div>
  <div
    className="App-container"
  >
    <div
      className="app-lists"
    >
      <ColumnList
        addTask={[Function]}
        items={Array []}
        title="To Do"
        updateTask={[Function]}
      />
      <ColumnList
        addTask={[Function]}
        items={Array []}
        title="Done"
        updateTask={[Function]}
      />
    </div>
  </div>
</div>
`;
